tests/docker-compose/docker-compose.yml

```diff
@@ -0,0 +1,22 @@
+services:
+  irods-server:
+    container_name: irods-server
+    image: wsinpg/ub-18.04-irods-4.2.11
+    restart: always
+    ports:
+      - "1247:1247"
+
+  irods-clients: # setup (symlinks etc.) required is documented in irods-client readme of containers repo
+    container_name: irods-clients
+    image: wsinpg/md-bullseye-irods-clients-4.2.11
+    volumes:
+      - $PWD/../..:$PWD/../..
+      - $PWD:$HOME/.irods/
+    environment:
+      - CLIENT_USER=$USER
+      - CLIENT_USER_ID=$UID # shell variable requires setting UID=${UID} at the beginning of the docker-compose command
+      - CLIENT_USER_HOME=$HOME
+      - IRODS_ENVIRONMENT_FILE=$PWD/irods_environment.json
```

Comment:
`$HOME` and `$PWD` should both be quoted

Reply:
Here's part of my local config for comparision:

```
      CLIENT_USER_ID: "${UID:? \nERROR: The UID environment variable is unset}"
      CLIENT_USER: "${USER:? \nERROR: The USER environment variable is unset}"
      CLIENT_USER_HOME: "${HOME}"
```

`UID` can't be set in bash (it can in some other shells) - for the others, it's nice to have warnings if some essentials are unset.